src/libultra_code/osRumblePak.c

```diff
@@ -22,7 +22,7 @@ s32 osSetRumble(unk_controller_t* arg0, u32 vibrate) {
         ((PIF_mempak_wr_t*)buf)->data[i + 2] = vibrate;
     }
 
-    _osCont_lastPollType = 0xfe; // last controller poll type?
+    __osContLastPoll = 0xfe; // last controller poll type?
```

Comment:
assuming that this is being renamed cause we know what it is, in which case the comment should be removed

Reply:
also i know you didnt add this, but that hex should probably be uppercase

Reply:
The official name for this is __osContLastCmd but we have been calling it "poll" instead of "cmd." Should I keep it this way?